trin-types/src/jsonrpc/request.rs

```diff
@@ -8,6 +8,9 @@ use crate::jsonrpc::params::Params;
 
 type Responder<T, E> = mpsc::UnboundedSender<Result<T, E>>;
 
+/// Portal network defines "content absent" response as "0x".
+pub const CONTENT_ABSENT: &str = "0x";
```

Comment:
How do we feel about moving this constant in a separate file `constants.rs` for example in `trin-types` crate?

Reply:
I thought about it, but did not see a benefit. It is a value inherent to the JSON-RPC, so thought it appropriate here. 
I guess it is good convention though, so I'll move it.